src/Admin/CitizenProjectDatagrid.php

```diff
@@ -26,13 +26,11 @@ public function __construct(DatagridInterface $decorated, CitizenProjectManager
     public function getResults()
     {
         if (!$this->cachedResults) {
-            /** @var CitizenProject[] $results */
```

Comment:
Why moving the phpdoc below after the initialization? Maybe you can just remove it.

Reply:
I don't know why I did that hahaha. In this context it is necessary to help the IDE.